kumascript/tests/macros/Deprecated.test.ts

```diff
@@ -1,4 +1,4 @@
-import { assert, itMacro, describeMacro } from "./utils";
+import { assert, itMacro, describeMacro } from "./utils.js";
```

Comment:
@queengooborg I feel like it's really weird that we have to import from `*.js`, even though the source is a TypeScript file. So I wonder if it's worth waiting for TypeScript 5.0 (due March 2023), which should introduce `moduleResolution: hybrid` that supports importing from `*.ts`. :/

Reply:
It is a little strange, I agree -- most other TypeScript projects seemed to have gone forward with using `.js` for now, and personally I don't think that we should wait another four months for ESM migration over something like this?

Reply:
Makes sense, you're right.